admin.go

```diff
@@ -167,6 +170,16 @@ func handleLoadConfig(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func handleStop(w http.ResponseWriter, r *http.Request) {
+	if r.Method == http.MethodPost {
+		log.Println("Shutting down...")
+		Stop()		
```

Comment:
While currently this function does not return any errors, might as well handle it anyway:

```go
if err := Stop(); err != nil {
    log.Printf("[ADMIN][ERROR] stopping: %v", err)
}
```

Reply:
done